src/main/java/com/sproutsocial/nsq/SingleNsqdBallenceStrategy.java

```diff
@@ -0,0 +1,70 @@
+package com.sproutsocial.nsq;
+
+import org.slf4j.Logger;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+/**
+ * A single NSQD ballence strategy will attempt to publish to the single known NSQD.  If that fails on a first attempt
+ * it will *block* for the failover duration (default 10 seconds) and attempt to reconnect.
+ */
+public class SingleNsqdBallenceStrategy extends BasePubSub implements BalanceStrategy {
+    private static final Logger logger = getLogger(SingleNsqdBallenceStrategy.class);
+    protected final ConnectionDetails connectionDetails;
+    private int failoverDurationSecs = 10;
+
+    public SingleNsqdBallenceStrategy(Client client, Publisher parent, String nsqd) {
+        super(client);
+        logger.warn("You are configured to use a singe NSQD balance strategy.  Please Read the manual and make sure you really want that.");
```

Comment:
haha, love that you added this warning. Can you be more specific about why a user might not want it? IE - "This means you will drop messages if your configured nsqd is unreachable or offline"

Reply:
Kind of a lot of details...  but give it a look